MAUI/ListView/viewappearance.md

```diff
@@ -0,0 +1,1208 @@
+---
+layout: post
+title: Appearance in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Appearance support in Syncfusion .NET MAUI ListView (SfListView) control and more.
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# Appearance in .NET MAUI ListView (SfListView)
+
+The `SfListView` allows customizing appearance of the underlying data, and provides different functionalities to the end-user.
+
+## Item template
+
+A template can be used to present the data in a way that makes sense for the application by using different controls. `SfListView` allows customizing appearance of view by setting the [ItemTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTemplate) property.
+
+## Data template selector
+
+The `SfListView` allows customizing appearance of each item with different templates based on specific constraints by using the `DataTemplateSelector`. You can choose a `DataTemplate` for each item at runtime based on the value of data-bound property using `DataTemplateSelector`.
```

Comment:
The `SfListView` allows customizing the appearance of each item with different templates based on specific constraints by using the `DataTemplateSelector`.

Reply:
I have modified the line as per the concerns provided.